### DISCUSSION TITLE
Error message on website after using PDF Generator

### DISCUSSION THREAD

**Discussion by HeSwartz** [code4recovery/12-step-meeting-list#1552] [Q&A]
We've just started using the PDF generator on our website ct-aa.org.  We just got this error message.  Has anyone ever gotten this before?
![Error Message](https://github.com/user-attachments/assets/1d17b8e1-9e38-4645-a227-b0f235860e82)

Because the site is down I can't confirm the software versions.

**Comment by joshreisner** [comment]
i'm not seeing that error, i checked a few hours ago as well
<img width="1609" alt="Screenshot 2024-10-26 at 4 16 51 PM" src="https://github.com/user-attachments/assets/dcf8380a-d995-4317-8615-bd92d2e77342">

i don't think using the PDF generator should add any special burden on the website. i'm able to generate a PDF of your meetings without any issues.

are you still seeing the error on your end? when i am dealing with a website that is experiencing intermittent issues i'll set up a free monitor at uptimerobot.com, this can tell you how often the site goes down and for how long

**Comment by HeSwartz** [comment]
Hi Josh.  Yes, the problem has been resolved. GoDaddy made a change to the CDN and the firewall GoDaddy placed in front of our site didn’t like the change.  The fact that it happened around the time we started using the PDF Generator was just a coincidence.  Thanks for the excellent service.  It is much better than most tech companies. Regards, Herb.
________________________________
